src/lib/stores/billing.ts

```diff
@@ -463,30 +463,36 @@ export async function checkForNewDevUpgradePro(org: Organization) {
     if (now - accountCreated < 1000 * 60 * 60 * 24 * 7) return;
     const isDismissed = !!localStorage.getItem('newDevUpgradePro');
     if (isDismissed) return;
-    if (now - accountCreated < 1000 * 60 * 60 * 24 * 37) {
-        headerAlert.add({
-            id: 'newDevUpgradePro',
-            component: newDevUpgradePro,
-            show: true,
-            importance: 1
-        });
+    // check if coupon already applied
+    try {
+        await sdk.forConsole.billing.getCoupon('appw50');
```

Comment:
why does this check use a hardcoded coupon code? is there a better way to see if a coupon or a list of coupons are already applied?

Reply:
This is for something else, for 
https://github.com/appwrite/console/blob/1d9a1c58b0452ad5a17844e593eb34cbd81c6cfc/src/lib/components/billing/alerts/newDevUpgradePro.svelte#L32-L33
Which also hard-codes the coupon code.

Reply:
lets use a constant instead with some comments.

Reply:
Updated to use constant